client/template.go

```diff
@@ -188,3 +201,39 @@ func (client *ApiClient) AssignTemplateToProject(id string, payload TemplateAssi
 func (client *ApiClient) RemoveTemplateFromProject(templateId string, projectId string) error {
 	return client.http.Delete("/blueprints/" + templateId + "/projects/" + projectId)
 }
+
+func (client *ApiClient) VariablesFromRepository(payload *VariablesFromRepositoryPayload) ([]ConfigurationVariable, error) {
+	b, err := json.Marshal(payload)
+	if err != nil {
+		return nil, err
+	}
+
+	var paramsInterface map[string]interface{}
+	if err := json.Unmarshal(b, &paramsInterface); err != nil {
+		return nil, err
+	}
+
+	params := map[string]string{}
+	for key, value := range paramsInterface {
+		if key == "githubInstallationId" {
+			params[key] = strconv.Itoa(int(value.(float64)))
+		} else if key == "sshKeyIds" {
+			sshkeys := []string{}
+			if value != nil {
+				for _, sshkey := range value.([]interface{}) {
+					sshkeys = append(sshkeys, "\""+sshkey.(string)+"\"")
```

Comment:
I think this is correct:
`...&sshKeyIds=["id1","id2"]&...`

Reply:
@sagilaufer1992 - I think this covers it, right?

```
data "env0_ssh_key" "ssh_key" {
  name = "coool"
}

resource "env0_template" "example_terragrunt" {
  name               = "example - Terragrunt - sshkey"
  description        = "Example template with Terragrunt version"
  repository         = "https://github.com/env0/templates"
  path               = "terragrunt/misc/null-resource"
  ssh_keys           = [data.env0_ssh_key.ssh_key]
  type               = "terragrunt"
  terragrunt_version = "0.35.0"
}
```

```
  # env0_template.example_terragrunt will be created
  + resource "env0_template" "example_terragrunt" {
      + description          = "Example template with Terragrunt version"
      + id                   = (known after apply)
      + is_bitbucket_server  = false
      + is_github_enterprise = false
      + is_gitlab_enterprise = false
      + name                 = "example - Terragrunt - sshkey"
      + path                 = "terragrunt/misc/null-resource"
      + repository           = "https://github.com/env0/templates"
      + ssh_keys             = [
          + {
              + "id"   = "d98a1783-c980-4b45-b47e-f0252dcab7d8"
              + "name" = "coool"
            },
        ]
      + terraform_version    = "0.15.1"
      + terragrunt_version   = "0.35.0"
      + type                 = "terragrunt"
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.
```

![image](https://user-images.githubusercontent.com/12767692/170884794-4ad766fb-26f6-44ee-9d36-bbaf3b5dbbe7.png)